awscli/argparser.py

```diff
@@ -16,6 +16,19 @@
 from difflib import get_close_matches
 
 
+HELP_BLURB = (
+    "To see help text, you can run:\n"
+    "\n"
+    "  aws help\n"
+    "  aws <command> help\n"
+    "  aws <command> <subcommand> help\n"
```

Comment:
You can also have more than one subcommand with the wait commands. Not sure if there is a better way to represent this. Maybe:
```py
"To see help text, you can run:\n"
"\n"
"  aws help\n"
"  aws <command> [<subcommand> ...] help\n"
```

Same comment applies about the usage